connector_prestashop/views/prestashop_backend_view.xml

```diff
@@ -0,0 +1,406 @@
+                            <field name="sale_team_id" />
+                            <field
+                                name="importable_order_state_ids"
+                                widget="many2many_tags"
+                            />
+                            <field name="discount_product_id" />
+                        </group>
+                        <group string="Stock">
+                            <field name="warehouse_id" />
+                            <field name="stock_location_id" />
+                            <field name="shipping_product_id" />
+                             <field name="product_qty_field" />
+                        </group>
+                        <group string="Accounting">
+                            <field name="refund_journal_id" />
+                            <field name="taxes_included" />
+                        </group>
+
+                    </group>
+                    <notebook attrs="{'invisible':[('state', 'in', ['draft'])]}">
+                        <page name="import" string="Imports">
+                            <p class="oe_grey oe_inline">
+                                By clicking on the buttons,
+                                you will initiate the
+                                synchronizations
+                                with PrestaShop.
+                                Note that the import or exports
+                                won't be done directly,
+                                they will create 'Jobs'
+                                executed as soon
+                                as possible.
+                            </p>
+                            <p class="oe_grey oe_inline">
+                                Once imported,
+                                some types of records,
+                                like the products
+                                or categories,
+                                need a manual review.
+                                You will find the list
+                                of the
+                                new records to review
+                                in the menu 'Connectors > Checkpoint'.
+                            </p>
+                            <group>
+                                <div>
+                                    <label
+                                        string="Import product categories and products"
+                                        class="oe_inline"
+                                        for="import_products_since"
+                                    />
+                                    <field
+                                        name="import_products_since"
+                                        class="oe_inline"
+                                        nolabel="1"
+                                    />
+                                </div>
+                                <button
```

Comment:
If you put button before the div of label, usability is improved:

Before:

![before](https://github.com/user-attachments/assets/f5ca6f40-328d-45ad-b5fa-92049f04144c)


After:

![after](https://github.com/user-attachments/assets/82ee44ca-3349-44ca-b7f8-845acd40bb34)